src/Redist/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.TensorFlow.Redist.proj

```diff
@@ -9,20 +9,12 @@
     <TensorFlowConfig Include="windows" FileExtension=".zip" FilesFromArchive="lib\tensorflow.dll;include\tensorflow\c\LICENSE" Runtime="win-x64"/>
     <TensorFlowConfig Include="linux" FileExtension=".tar.gz"
                       FilesFromArchive="lib\libtensorflow.so;
-                                        lib\libtensorflow.so.$(TensorFlowMajorVersion);
```

Comment:
Are all these dynamic libraries not necessary then?

Reply:
The binaries deleted are identical to the ones remaining. They are symlinked in the tar file. But when they get to the nuget package, they become copies. 
This change keeps only the ones that are used.